onnxruntime/python/tools/transformers/onnx_exporter.py

```diff
@@ -202,7 +202,7 @@ def optimize_onnx_model(onnx_model_path, optimized_model_path, model_type, num_a
 
         if Precision.FLOAT16 == precision:
             opt_model.convert_model_float32_to_float16()
-        opt_model.save_model_to_file(optimized_model_path, use_external_data_format)
+        opt_model.save_model_to_file(optimized_model_path)
```

Comment:
What if a model is really large so optimized model shall use external data format?

Reply:
Yes, our goal is to add it back. 
But currently it does not behave expectedly, and causing issue for more models than having it. I think it may take longer time to investigate it.

Reply:
This looks like introducing new problem so it is not a clean fix.

How about changing external data flag for those models (or disable those models) having such issue?